src/librustdoc/clean/types.rs

```diff
@@ -1407,6 +1405,36 @@ impl PrimitiveType {
         }
     }
 
+    crate fn get_symbol(self) -> Symbol {
```

Comment:
Heh, I also had to add this for https://github.com/rust-lang/rust/pull/80090. Can you call it `as_sym` for consistency with `as_str`?

Reply:
Sure!